frontend/src/page/Video.tsx

```diff
@@ -1,35 +1,36 @@
 import React from "react";
-import { graphql, useLazyLoadQuery } from "react-relay/hooks";
+import { graphql, loadQuery, usePreloadedQuery } from "react-relay/hooks";
+import type { PreloadedQuery } from "react-relay/hooks";
 
 import type { VideoQuery } from "../query-types/VideoQuery.graphql";
+import { environment as relayEnv } from "../relay";
 import { Player } from "../ui/Player";
 import type { Route } from "../router";
 import { Root } from "../layout/Root";
 
 
-export const VideoRoute: Route<Record<string, string>> = {
+export const VideoRoute: Route<PreloadedQuery<VideoQuery>> = {
```

Comment:
Looking at this, these `*Route` components all look pretty similar/boilerplate-y. I wonder if one couldn't trim that down a bit. Have you thought about that? But it's not super important, of course.

Reply:
I have but I figured it's not that important. Additionally, the route definitions get more different in follow up PRs. Again, this PR is preparation for stuff mostly.

Reply:
Makes sense. :shrug: